### PR TITLE
Grammatically updated the docs

### PR DESCRIPTION
**Pull request by vaibhav0806**
Small grammatical updates to the AnkiDroid docs.

### CODE REVIEW

File: manual.asc [hunk 1]
```diff
@@ -80,14 +80,14 @@ Note that the currently selected deck is highlighted with a grey background, and
 === Other Deck Actions
 Long tapping on a deck will show a list of other actions available to perform on that deck:
 
-Rename deck :: Use this option to rename a deck
+Rename deck :: Use this option to rename a deck.
```
Comment (mikehardy) [inline manual.asc:83]: Hi there! These line endings are not in agreement with each other. You used colons above and periods here. I propose using no line endings thoughtoug, as was the case before

Comment (vaibhav0806) [reply manual.asc:83]: Alright, I will make that change!

**Review by mikehardy — CHANGES_REQUESTED** [review]
I think since they are sentence fragments, no line endings are needed, that removes the vast majority of this PR's diff
I like the changes on line 45 and 56 though, they feel better in consideration of the rest of the prompts. If it were just those two lines I would merge

**Comment by vaibhav0806** [comment]
> I think since they are sentence fragments, no line endings are needed, that removes the vast majority of this PR's diff I like the changes on line 45 and 56 though, they feel better in consideration of the rest of the prompts. If it were just those two lines I would merge



> I think since they are sentence fragments, no line endings are needed, that removes the vast majority of this PR's diff I like the changes on line 45 and 56 though, they feel better in consideration of the rest of the prompts. If it were just those two lines I would merge

Oka

File: manual.asc (hunk 1 above)
Comment (vaibhav0806) [reply manual.asc:83]: Alright, I will keep the changes on line number 45 and 56 and undo the rest of them!

**Pull request closed without merging**